Tinylog/Views/Cells/TextFieldCell.swift

```diff
@@ -9,6 +9,8 @@
 import UIKit
 
 final class TextFieldCell: UITableViewCell, UITextFieldDelegate, TextFieldCellDelegate {
+    
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/sger/Tinylog/blob/03eace86b303bd47c13e99b11bd2b1214da5c26b/Tinylog/Views/Cells/TextFieldCell.swift#L12"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>TextFieldCell.swift:12</code></td>
    </tr>
  </tbody>
</table>